server/filters/spacing-classes.js

```diff
@@ -1,6 +1,11 @@
-export default function spacingClasses(sizes, properties) {
-  return Object.keys(sizes).map(key => {
-    const size = sizes[key];
+import { getSizesAsObject } from './utils';
+
+export default function spacingClasses(sizes, properties, fillIn = true) {
```

Comment:
`fillIn` explanation:

In the main, we'd want to be able to write `{{ {s:2} | spacingClasses }}` and have all of the breakpoints populated with that size (i.e. equivalent to writing `{{ {s:2, m:2, l:2, xl:2} | spacingClasses }}`, but there are occasions when we only want to apply spacing classes at one or two breakpoints, so this allows us to override that behaviour.